src/lib/components/LeftSidebarNav.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+    import Link from './Link.svelte';
+    import PlayIcon from '$lib/elements/icons/Play.svelte';
+    import UploadIcon from '$lib/elements/icons/Upload.svelte';
+    import MusicNoteIcon from '$lib/elements/icons/MusicNote.svelte';
+    import { currentUser } from '$lib/stores/currentUser';
+</script>
+
+<div class="flex flex-col gap-4 mt-8 font-semibold text-lg">
+    <Link
+        href={`/${$currentUser?.npub}/listen`}
```

Comment:
Feels strange to me to nest the listen route under the logged in user's `npub`. What's the thinking there? Playlists and Uploaded tracks make sense.